common/app/rendering/core/JavascriptProps.scala

```diff
@@ -0,0 +1,22 @@
+package rendering.core
+
+import conf.Configuration
+import play.api.libs.json.{Json, Writes}
+
+case class JavascriptPropsConfig(
+  bundleUrl: String = conf.Static("javascripts/ui.bundle.browser.js"),
+  polyfillioUrl: String =
+    if (conf.switches.Switches.PolyfillIO.isSwitchedOn) common.Assets.js.polyfillioUrl
+    else conf.Static("javascripts/vendor/polyfillio.fallback.js"),
+  beaconUrl: String = Configuration.debug.beaconUrl
+)
+case class JavascriptProps(
+  config: JavascriptPropsConfig = JavascriptPropsConfig()
```

Comment:
I think it would make more sense to have no default value for the `JavascriptPropsConfig` and `JavascriptProps` and have a `default` function that return an instance of `JavascriptProps` with the values you specified?
It would be more explicit to call `JavascriptProps.default` rather than `JavascriptProps()`